decoders/connector/atomsenses/as-204/v1.0.0/payload-config.jsonc

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "../../../../../schema/connector_details.json",
+  "description": "../description.md",
+  "install_text": "AS-204 is a compact indoor ambience monitoring sensor for measurement of temperature, humidity, CO2 concentration, NH3, H2S. Sensor data is transmitted using LoRaWAN® technology.\n\n**Features**\n*  Support batteries or DC power supply\n* Compliant with standard LoRaWAN® gateways and network servers \n\n**Datasheet**\n* https://www.atomsenses.com/wp-content/uploads/2024/07/AS-204.pdf\n",

```

Comment:
Please remove external link they aren't allowed as they may become broken in the future.